test/fixtures/experimental/class-private-properties/asi-failure-literal/actual.js

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  #"p" = x
+  #2 = y
```

Comment:
It'd be nice to break this out into a separate test file, so you can check that each of them is a syntax error.